Events/Messages/MessageCreate.ts

```diff
@@ -1,11 +1,11 @@
+import { Message } from "discord.js"
 import { DiscordClient } from "../../Utils/DiscordClient"
 
 export const name = "messageCreate"
-export function execute(message, client: DiscordClient) {
-	if(blacklist(message.content)) {
-		message.delete()
-		message.channel.send({content: `:x: Your message contained a blacklisted phase ${message.member.tag}`})
-	}
+export function execute(message: Message, client: DiscordClient) {
+	if(!blacklist(message.content)) return;
+	message.delete()
+	message.channel.send({content: `:x: Your message contained a blacklisted phase <@!${message?.author.id}>!`})
```

Comment:
Why is `?.` used here? According to the type of message, it has to be defined, right?